Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InfiniteCanvas/InfiniteCanvasPage.xaml.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Microsoft.Toolkit.Uwp.Helpers;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    /// <summary>
+    /// InfinteCanvas sample page.
+    /// </summary>
+    public sealed partial class InfiniteCanvasPage : Page, IXamlRenderListener
+    {
+        private const string InfiniteCanvasFileName = "infiniteCanvasFile.txt";
+        private InfiniteCanvas _infiniteCanvas;
+
+        public InfiniteCanvasPage()
+        {
+            InitializeComponent();
+        }
+
+        public void OnXamlRendered(FrameworkElement control)
+        {
+            _infiniteCanvas = control.FindChildByName("canvas") as InfiniteCanvas;
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            base.OnNavigatedTo(e);
+
+            Shell.Current.RegisterNewCommand("Export & Save", async (sender, args) =>
+            {
+                var json = _infiniteCanvas.ExportAsJson();
+                await StorageFileHelper.WriteTextToLocalFileAsync(json, InfiniteCanvasFileName, Windows.Storage.CreationCollisionOption.ReplaceExisting);
+            });
+
+            Shell.Current.RegisterNewCommand("Import and Load", async (sender, args) =>
```

Comment:
Can you update these commands to save and load files outside of the app. They should ask the user where to save or what file to import.